src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -177,11 +167,9 @@ case class JesPendingExecutionHandle(backendCall: JesBackendCall,
                                      run: Run,
                                      previousStatus: Option[RunStatus]) extends ExecutionHandle {
   override val isDone = false
-
-  override def result = FailedExecution(new IllegalStateException)
+  override val result = NonRetryableExecution(new IllegalStateException)
```

Comment:
I kind of like the `IllegalStateException` because it describes accurately what happens if we get there. What about just `override val result = throw new IllegalStateException("JesPendingExecutionHandle cannot yield a result")` ?